src/main/java/com/ss/training/utopia/agent/service/AgentService.java

```diff
@@ -17,6 +17,33 @@
     @Autowired
     BookingDAO bookingDAO;
 
+        public void createBooking(Booking booking) {
+        // HOW TO DO A STRIPE PURCHASE
+
+        // STORE A STRIPE KEY
+        // Stripe.apiKey = "sk_test_4eC39HqLyjWDarjtT1zdp7dc";
+
+        // GET THE PRICE FROM FLIGHTS TABLE
+        // float price = flightDAO.getByFlightId().getPrice();
+
+        // CREATE A CHARGE OBJECT
+        // `source` is obtained with Stripe.js; see https://stripe.com/docs/payments/accept-a-payment-charges#web-create-token
+        // Map<String, Object> params = new HashMap<>();
+        // params.put("amount", 2000);
+        // params.put("currency", "usd");
+        // params.put("source", "tok_mastercard");
+        // params.put(
+        //   "description",
+        //   "My First Test Charge (created for API docs)"
+        // );
+
+        // Charge charge = Charge.create(params);
+
+        // Handle the Stripe request
+
+        bookingDAO.save(booking);
```

Comment:
I think there should be some exception handling here in case the database operation fails.